Sources/SwiftTwitch/Classes/Results/Users/UpdateUserData.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UpdateUserData.swift
+//  SwiftTwitch
+//
+//  Created by Christopher Perkins on 12/31/18.
+//
+
+import Foundation
+
+/// `UpdateUserData` is a class that encapsulates all of the information returned by the
+/// `Update User` call of the New Twitch API. This includes:
+/// * The data of the user that was updated
+public struct UpdateUserData: Codable {
+    
+    /// `userData` contains the data of the users that were retrieved from a `Update User` API call.
+    public let userData: [UserData]
```

Comment:
Different property name vs web request key name, need to change variable name or do custom Codable initialization